spec/games_factory_spec.rb

```diff
@@ -0,0 +1,30 @@
+#do we need to require lib/games?  Why not just games_factory
+# Should this be `Game` or `Games`
+require './lib/games'
+require './lib/games_factory'
+require 'pry'
+
+RSpec.describe GamesFactory do
+  before do
+        # if @file_path is an attribute, why is it in the before block?  
+    @file_path = './data/game_fixture.csv'
```

Comment:
We need to instantiate @file_path to give it as parameter for the initialize.

Reply:
Ah, right.  Okay.  Thank you for spelling that out. 

I got final changes in with changing `Games` to `game` since we're making Game objects.